configure

```diff
@@ -130,6 +130,8 @@ DSC_SCRIPT_PATH=$DSC_SCRIPT_PATH
 
 BUILD_DEBUG=$BUILD_DEBUG
 DEBUG_FLAGS=$DEBUG_FLAGS
+# DSC cannot build with -Werror
+ENABLE_WERROR=0
```

Comment:
I don't think it's necessary to switch the includes of the top-level `Makefile`, as it doesn't affect what happens here, and I'd rather leave up to @johnkord to make such a possibly dangerous change.